sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.institutions.controller.exportReport.e2e-spec.ts

```diff
@@ -0,0 +1,283 @@
+              "Offering Intensity": firstSavedOffering.offeringIntensity,
+              "Course Load": firstSavedOffering.courseLoad.toString(),
+              "Delivery Type": firstSavedOffering.offeringDelivered,
+              "WIL Component": firstSavedOffering.hasOfferingWILComponent,
+              "WIL Component Type": firstSavedOffering.offeringWILType ?? "",
+              "Start Date": firstSavedOffering.studyStartDate,
+              "End Date": firstSavedOffering.studyEndDate,
+              "Has Study Breaks":
+                (!firstSavedOffering.lacksStudyBreaks).toString(),
+              "Actual Tuition":
+                firstSavedOffering.actualTuitionCosts.toString(),
+              "Program Related Costs":
+                firstSavedOffering.programRelatedCosts.toString(),
+              "Mandatory Fees": firstSavedOffering.mandatoryFees.toString(),
+              "Exceptional Expenses":
+                firstSavedOffering.exceptionalExpenses.toString(),
+              "Offering Type": firstSavedOffering.offeringType,
+              Status: firstSavedOffering.offeringStatus,
+              "Funded Weeks":
+                firstSavedOffering.studyBreaks.totalFundedWeeks.toString(),
+              "Total Applications": "",
+            }),
+            expect.not.objectContaining({
+              "Institution Location Code": collegeCLocation.institutionCode,
+              Program: thirdSavedOffering.educationProgram.name,
+              "SABC Program Code":
+                thirdSavedOffering.educationProgram.sabcCode ?? "",
+              "Offering Name": thirdSavedOffering.name,
+              "Year of Study": thirdSavedOffering.yearOfStudy.toString(),
+              "Offering Intensity": thirdSavedOffering.offeringIntensity,
+              "Course Load": thirdSavedOffering.courseLoad.toString(),
+              "Delivery Type": thirdSavedOffering.offeringDelivered,
+              "WIL Component": thirdSavedOffering.hasOfferingWILComponent,
+              "WIL Component Type": thirdSavedOffering.offeringWILType ?? "",
+              "Start Date": thirdSavedOffering.studyStartDate,
+              "End Date": thirdSavedOffering.studyEndDate,
+              "Has Study Breaks":
+                (!thirdSavedOffering.lacksStudyBreaks).toString(),
+              "Actual Tuition":
+                thirdSavedOffering.actualTuitionCosts.toString(),
+              "Program Related Costs":
+                thirdSavedOffering.programRelatedCosts.toString(),
+              "Mandatory Fees": thirdSavedOffering.mandatoryFees.toString(),
+              "Exceptional Expenses":
+                thirdSavedOffering.exceptionalExpenses.toString(),
+              "Offering Type": thirdSavedOffering.offeringType,
+              Status: thirdSavedOffering.offeringStatus,
+              "Funded Weeks":
+                thirdSavedOffering.studyBreaks.totalFundedWeeks.toString(),
+              "Total Applications": "",
```

Comment:
Would it not work in the same way if only "Institution Location Code" and "Program" are used?